Userland/Libraries/LibCore/CMakeLists.txt

```diff
@@ -38,3 +38,7 @@ endif()
 
 serenity_lib(LibCore core)
 target_link_libraries(LibCore ${LIBRARY_LIBC} ${LIBRARY_LIBCRYPT})
+
+if(BUILD_LAGOM)
+    target_link_libraries(LibCore ${LIBRARY_LIBPTHREAD})
```

Comment:
Why is that necessary and should we perhaps also do it for SerenityOS?

Reply:
IIRC this comes from AK's StackInfo, which has a couple of calls to pthread on Linux.